cmake/common.cmake

```diff
@@ -61,6 +62,15 @@ macro(nbl_create_executable_project _EXTRA_SOURCES _EXTRA_OPTIONS _EXTRA_INCLUDE
 		
 		if(NBL_DYNAMIC_MSVC_RUNTIME)
 			set_property(TARGET ${EXECUTABLE_NAME} PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
+			
+			if(WIN32 AND MSVC)
+				set(_NABLA_OUTPUT_DIR_ "${NBL_ROOT_PATH_BINARY}/src/nbl/$<CONFIG>/devshgraphicsprogramming.nabla")
+				
+				target_link_options(${EXECUTABLE_NAME} PUBLIC "/DELAYLOAD:$<TARGET_FILE_NAME:Nabla>")
+				target_compile_definitions(${EXECUTABLE_NAME} PUBLIC 
+					_NABLA_DLL_NAME_="$<TARGET_FILE_NAME:Nabla>";_NABLA_OUTPUT_DIR_="${_NABLA_OUTPUT_DIR_}";_NABLA_INSTALL_DIR_="${CMAKE_INSTALL_PREFIX}"
```

Comment:
`_NABLA_DLL_NAME_`  and `_NABLA_INSTALL_DIR_` should go into the generated `nabla.h`

P.S. it would be a cool install step trick to only paste `_NABLA_INSTALL_DIR_` into `nabla.h` while its being installed ;)

Reply:
hmm, lemme try something then please

Reply:
done, installed `nabla.h` includes `BuildConfigOptions.h` that includes `define.h` having those 2 defines

Reply:
> @AnastaZIuk can you remove all the manifest crap from before then?

removed